boom/tests/test_boom.py

```diff
@@ -34,11 +49,17 @@ def handle(self, env, start_response):
         elif env['PATH_INFO'] == '/reset':
             self.numcalls = 0
             start_response('200 OK', [('Content-Type', 'text/plain')])
-            return ['numcalls set to zero']
+            if PY3:
+                return ['numcalls set to zero'.encode('latin-1')]
+            else:
+                return ['numcalls set to zero']
         else:
-            start_response(
-                '404 Not Found', [('Content-Type', 'text/plain')])
-            return ['%s' % env['PATH_INFO']]
+            start_response('404 Not Found',
+                           [('Content-Type', 'text/plain')])
+            if PY3:
+                return [env['PATH_INFO'].encode('latin-1')]
```

Comment:
Part of the gevent changes with python3, it requires bytes instead of strings, I used latin-1 because that's the encoding they use internally on their tests.